Makefile

```diff
@@ -54,10 +54,16 @@ release: lint test
 	mkdir release
 	GOOS=darwin GOARCH=amd64 go build -ldflags="-X main.VERSION=${VERSION}" -o release/$(BINARY)-darwin-amd64 github.com/pusher/oauth2_proxy
 	GOOS=linux GOARCH=amd64 go build -ldflags="-X main.VERSION=${VERSION}" -o release/$(BINARY)-linux-amd64 github.com/pusher/oauth2_proxy
+	GOOS=linux GOARCH=arm64 go build -ldflags="-X main.VERSION=${VERSION}" -o release/$(BINARY)-linux-arm64 github.com/pusher/oauth2_proxy
+	GOOS=linux GOARCH=arm GOARM=6 go build -ldflags="-X main.VERSION=${VERSION}" -o release/$(BINARY)-linux-armv6 github.com/pusher/oauth2_proxy
```

Comment:
Am I right in thinking that if we changed this to `GOARM=7` then it wouldn't run on original Pi's and Pi zeros whereas currently it runs on all Pi's?

Reply:
I believe that is correct. It allows new v7 only asm instructions to be used for the build that the zero doesn't have. v7 is a superset of v6.

Reply:
Indeed. 
With go and dep dependencies I decided to just leave this at armv6 for now.
As it builds and runs on my armv7 I figure it's safe to start with.